server/engine/telemetry.go

```diff
@@ -27,11 +28,29 @@ func GetEvent(propertiesMap analytics.Properties) string {
 func BuildSegmentPropertiesMap(db *gorm.DB) analytics.Properties {
 
 	var propertiesMap = analytics.Properties{}
+	retriesMap := make(map[string]interface{})
+	errorsMap := make(map[string]interface{})
 	var metricData []model.Telemetry
 	db.Find(&metricData)
+	// Error Details and Number of retries will be granular at the step level hence, each error and retry will be mapped to a step
+	// in a nested JSON like format
+	// Rest of the metrics are granular at the deployment level
 	for _, data := range metricData {
-		propertiesMap[string(data.MetricName)] = fmt.Sprintf("%v", data.MetricValue)
+		fmt.Printf("%+v\n", data)
```

Comment:
This seems like a left over from development.

Reply:
Removed this